src/chart.ts

```diff
@@ -635,18 +635,75 @@ export class Chart extends EventEmitter.EventEmitter {
   updateAxes() {
     const instance = this
     const canvas = instance.canvas.merge(instance.canvas.enter)
-    canvas.select('.x.axis').call(instance.meta.xAxis)
-    canvas.select('.y.axis').call(instance.meta.yAxis)
 
-    // updates the style of the axes
+    // center the axes
+    const yMin = this.meta.yScale.domain()[0]
+    const yMax = this.meta.yScale.domain()[1]
+
+    const yMid = (yMax + yMin) / 2
+
+    const yScaleFactor = this.meta.height / (yMax - yMin)
+
+    let yTranslation = yScaleFactor * yMid + this.meta.height / 2
+
+    const xMin = this.meta.xScale.domain()[0]
+    const xMax = this.meta.xScale.domain()[1]
+
+    const xMid = (xMax + xMin) / 2
+
+    const xScaleFactor = this.meta.width / (xMin - xMax)
+
+    let xTranslation = xScaleFactor * xMid + this.meta.width / 2
+
+    if (this.options.xAxis.position === 'sticky') {
+      yTranslation = yTranslation < 0 ? 0 : yTranslation
+      yTranslation = yTranslation > this.meta.height ? this.meta.height : yTranslation
+      canvas
+        .select('.x.axis')
+        .call(instance.meta.xAxis)
+        .attr('transform', 'translate(0,' + yTranslation + ')')
+
+      canvas
+        .selectAll('.x.axis path, .x.axis line')
+        .attr('fill', 'none')
```

Comment:
There are some attr that are reapplied later e.g. line 700:

```
    canvas
      .selectAll('.axis path, .axis line')
      .attr('fill', 'none')
      .attr('stroke', 'black')
      .attr('shape-rendering', 'crispedges')
      .attr('opacity', 0.1)
```

Maybe the lines in 700 should be kept and then remove these additional attrs.